### PR TITLE
fix failing test

### PR DESCRIPTION
**Pull request by nurali-techie**
1) Removed failing test - https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5539/rehearse-5539-pull-ci-codeready-toolchain-registration-service-master-test/3

Commit: https://github.com/codeready-toolchain/registration-service/pull/53/commits/3a0291517c494522c0c9fe54c6911719bbddc775

2) Fix Data Race

Commit: https://github.com/codeready-toolchain/registration-service/pull/53/commits/9c378c2b242fbb159d3b34c30ad80b9a95537732

3) Fix Typo - https://github.com/codeready-toolchain/registration-service/pull/51#discussion_r338405705

Commit: https://github.com/codeready-toolchain/registration-service/pull/53/commits/144a3681082ef5cac551109d696fc632ae1eba8c

Fixes: https://jira.coreos.com/browse/CRT-320

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -5,7 +5,7 @@ COV_DIR = $(OUT_DIR)/coverage
 test: generate
 	@echo "running the tests without coverage..."
 	# TODO enable -race flag, there are some DATA RACE need to be fixed
```
Comment (xcoulon) [inline make/test.mk]: you can remove this statement now

File: pkg/auth/defaultmanager_test.go [hunk 2]
```diff
@@ -127,18 +131,23 @@ func (s *TestDefaultManagerSuite) TestKeyManagerDefaultTokenParser() {
 	})
 
 	s.Run("parallel threads", func() {
+		s.T().Logf("VN:start")
```
[the remark this answers is not in the document]
Comment (nurali-techie) [reply pkg/auth/defaultmanager_test.go]: Done https://github.com/codeready-toolchain/registration-service/pull/53/commits/18f3d37ea44b15fdeb2ee0ce88ba6f520519bb1f

[the remark this answers is not in the document]
Comment (nurali-techie) [reply pkg/auth/defaultmanager_test.go]: Done https://github.com/codeready-toolchain/registration-service/pull/53/commits/396bbf4b9d28861a6cb93b3c1fd9950105c3617e

File: make/test.mk (hunk 1 above)
Comment (nurali-techie) [reply make/test.mk]: Done https://github.com/codeready-toolchain/registration-service/pull/53/commits/396bbf4b9d28861a6cb93b3c1fd9950105c3617e

**Comment by nurali-techie** [comment]
@xcoulon @MatousJobanek thanks for review :-)

**Pull request merged**